srcs/clients/client/include/Client.hpp

```diff
@@ -15,7 +15,7 @@
 #include "Request.hpp"
 #include "Response.hpp"
 
-#define RECEIVE_LEN 1000
+#define RECEIVE_LEN 1460 * 2
```

Comment:
TCP (Transmission Control Protocol): TCP 패킷의 경우, TCP 세그먼트의 데이터 부분의 최대 크기는 "Maximum Segment Size (MSS)"로 결정되며, 일반적으로 Ethernet 환경에서는 1460 바이트입니다. 이 값은 IP 헤더와 TCP 헤더 크기를 고려하여 결정됩니다.

Reply:
https://www.ibm.com/docs/ko/aix/7.1?topic=tuning-tcp-maximum-segment-size

```suggestion
#define RECEIVE_LEN 1460
```

이렇게 바꾸셔야 합니다.

Reply:
1패킷당 들어올 수 있는 실제 데이터는 최소 1380 최대 1460으로 알고 있습니다.
TCP 패킷 및 IP 패킷 모두 최소 20 바이트 최대 60바이트의 크기를 가집니다.

Reply:
좋은 지적감사합니다.